src/core/ext/transport/cronet/transport/cronet_transport.cc

```diff
@@ -555,6 +555,19 @@ static void on_response_headers_received(
   for (size_t i = 0; i < headers->count; i++) {
     if (0 == strcmp("grpc-status", headers->headers[i].key)) {
       on_response_trailers_received(stream, headers);
+      /* Do an extra read for a trailer-only stream with grpc_status = 0
```

Comment:
Looks like this block is a simple repeat of the block below. Can we put them in a single function?

Reply:
Done